app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -240,6 +234,19 @@ class HomeFragment : Fragment() {
         return view
     }
 
+    /**
+     * The [SessionControlView] is forced to update with our current state when we call [HomeFragment.onCreateView]
```

Comment:
nit: why do we need to draw everything at once? Make it clear this is a perf issue